public/conduct.jade

```diff
@@ -0,0 +1,30 @@
+div.jsla-toplevel-container.jsla-content-page
```

Comment:
does it need a redirect? it looked like it built it based on path. the change on line 1 is only that it's a 'new' file. i only changed the filename.